.github/labels.json

```diff
@@ -39,6 +39,8 @@
 	{ "name": "t⁴ ⋅ regression", "color": "#0052cc" },
```

Comment:
I think you can actually remove the first two about incomplete/removed templates now!

Reply:
People can still edit manually or use a blank template so they still might be useful?

Reply:
I guess you are right